### PR TITLE
Finish up frontend for groups

### PR DESCRIPTION
**Pull request by SimonBackx**

### CODE REVIEW

**Comment by SimonBackx** [comment]
Still needs some polishing, but for now going to work on the dashboard

File: .vscode/settings.json [hunk 1]
```diff
@@ -5,5 +5,13 @@
...
     },
-    "go.useLanguageServer": true
+    "go.useLanguageServer": true,
+    "vetur.format.options.tabSize": 4,
+    "prettier.tabWidth": 4,
+    "prettier.packageManager": "yarn",
+    "editor.formatOnSave": true,
+    "javascript.format.enable": false,
+    "vetur.format.defaultFormatter.html": "prettier",
+    "prettier.printWidth": 120,
```
Comment (AndreasBackx) [inline .vscode/settings.json]: Could this be set to 100? I prefer keeping everything under 80 lines, but use it as a soft limit with 100 being a strict limit. It makes everything a lot more readable.

File: frontend/create-group/package.json [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+        "@vue/eslint-config-typescript": "^4.0.0",
+        "eslint": "^5.16.0",
+        "eslint-plugin-vue": "^5.0.0",
+        "sass": "^1.23.7",
+        "sass-loader": "^8.0.0",
+        "typescript": "~3.5.3",
+        "vue-template-compiler": "^2.6.10"
+    },
+    "eslintConfig": {
+        "no-console": "off"
```
Comment (AndreasBackx) [inline frontend/create-group/package.json]: Could this be moved a global eslint/tslint config file so it is enabled for all of the frontend services?

File: frontend/create-group/src/components/EditGroupDetail.vue [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        <p>Dit is een test!</p>
+        <p>Dit is een test!</p>
+        <p>Dit is een test!</p>
+        <p>Dit is een test!</p>
+        <p>Dit is een test!</p>
+        <p>Dit is een test!</p>
+        <p>Dit is een test!</p>
+        <p>Dit is een test!</p>
+        <p>Dit is een test!</p>
+        <p>Dit is een test!</p>
```
Comment (AndreasBackx) [inline frontend/create-group/src/components/EditGroupDetail.vue]: Nice.

File: frontend/create-group/src/components/Steps.vue [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                    <GeneralStep v-on:next="goNext()"></GeneralStep>
+                </div>
+                <div v-if="step == 2" key="group-settings">
+                    <GroupSettingsStep v-on:next="goNext()"></GroupSettingsStep>
+                </div>
+                <div v-if="step == 3" key="groups">
+                    <GroupsStep v-on:next="goNext()"></GroupsStep>
+                </div>
+            </transition>
+        </main>-->
```
Comment (AndreasBackx) [inline frontend/create-group/src/components/Steps.vue]: Remove?

File: frontend/create-group/src/components/Steps.vue [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        NavigationController
+    }
+})
+export default class Steps extends Vue {
+    step: number = 1;
+    firstStep = new ComponentWithProperties(GeneralStep, {});
+    @Ref("navigationController") readonly navigationController!: NavigationController;
+
+    goBack() {
+        //this.step--;
```
Comment (AndreasBackx) [inline frontend/create-group/src/components/Steps.vue]: Remove?

File: frontend/create-group/src/components/steps/GroupSettings.vue [hunk 6]
```diff
@@ -20,10 +20,9 @@
                     <option>Scouts & Gidsen Vlaanderen</option>
```
Comment (AndreasBackx) [inline frontend/create-group/src/components/steps/GroupSettings.vue]: `&amp;`?

File: frontend/shared/components/inputs/SegmentedControl.vue [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+                text-overflow: ellipsis;
+                opacity: 1;
+                overflow: hidden;
+                word-wrap: none;
+                white-space: nowrap;
+                transition: opacity 0.25s;
+            }
+
+            &::after {
+                //background: white;
```
Comment (AndreasBackx) [inline frontend/shared/components/inputs/SegmentedControl.vue]: Remove?

File: frontend/shared/components/layout/ModalStackComponent.vue [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+    &.fade-leave-active,
+    &[data-extended-enter="true"] {
+        position: fixed;
+        transition: opacity 0.3s;
+
+        & > div {
+            transition: opacity 0.3s, transform 0.3s;
+        }
+    }
+    &.fade-enter, &.fade-leave-to /* .fade-leave-active below version 2.1.8 */ {
```
Comment (AndreasBackx) [inline frontend/shared/components/layout/ModalStackComponent.vue]: Below version 2.1.8?

File: frontend/shared/components/layout/Popup.vue [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+    &.fade-leave-active,
+    &[data-extended-enter="true"] {
+        position: fixed;
+        transition: opacity 0.3s;
+
+        & > div {
+            transition: opacity 0.3s, transform 0.3s;
+        }
+    }
+    &.fade-enter, &.fade-leave-to /* .fade-leave-active below version 2.1.8 */ {
```
Comment (AndreasBackx) [inline frontend/shared/components/layout/Popup.vue]: Idem.

File: .vscode/settings.json (hunk 1 above)
Comment (SimonBackx) [reply .vscode/settings.json]: No, the tab size is 4 spaces. When you write HTML, you almost immediately reach 80 characters and the HTML becomes unreadable.

File: frontend/create-group/package.json (hunk 2 above)
Comment (SimonBackx) [reply frontend/create-group/package.json]: Check other branch

**Pull request merged**